webapp/channels/src/components/post_view/post_message_preview/index.ts

```diff
@@ -43,7 +43,7 @@ function makeMapStateToProps() {
         let user = null;
         let embedVisible = false;
         let channelDisplayName = ownProps.metadata.channel_display_name;
-        const previewPost = getPost(state, ownProps.metadata.post_id) || ownProps.previewPost;
+        const previewPost = getPost(state, ownProps.metadata.post_id);
```

Comment:
If we are no longer using the previewPost coming from the props, and we are overwriting it, shouldn't we remove it? Also, I am worried on who used to pass the previewPost and how, to make sure they also don't have the post information.

And finally... what if the post is not in the store? (e.g. a post from a channel you haven't navigated to yet)